Remotion/Web/Development.WebTesting/ScreenshotCreation/Fluent/FluentScreenshotElement.cs

```diff
@@ -55,6 +58,19 @@ public class FluentScreenshotElement<T> : IFluentScreenshotElement<T>, IFluentSc
       _minimumElementVisibility = minimumElementVisibility;
     }
 
+    public FluentScreenshotElement<T> ScrollIntoView ()
```

Comment:
Hmm... Does it work if we do this as an extension method on `IFluentScreenshotElement<T> where ElementScope`?
Or, if that doesn't work, an exntesion method on `IFluentScreenshotElementWithCovariance<ElementScope>`?

That way, we don't need the guard condition, which, I believe, would make things cleaner.

Reply:
This works. That way, the API change with a separate task + ticket would not be necessary.

Reply:
okay, so we reduce the change to just adding an extension method, which is small enough to fit into the current PR. Hmm... I can see the logic in this.